website/docs/docs/build/data-tests.md

```diff
@@ -70,6 +70,30 @@ The name of this test is the name of the file: `assert_total_payment_amount_is_p
 
 Singular data tests are easy to write—so easy that you may find yourself writing the same basic structure over and over, only changing the name of a column or model. By that point, the test isn't so singular! In that case, we recommend...
 
+<VersionBlock firstVersion="1.9">
+
+### Document singular tests
+
+Previously, documenting a data test such as a singular data test would result in a parsing error. However, in [Versionless](/dbt-versions/upgrade-dbt-version-in-cloud#versionless), you can set descriptions in your YAML files to document singular data tests.
```

Comment:
I don't think we need to callout previous behavior. Instead, a callout that this is available in versionless and will be available in dbt Core `v1.9` and then just a description of the feature.